datastore/blob/azure_storage_api/__init__.py

```diff
@@ -112,24 +112,24 @@ async def get_blob(container_client, blob_name):
         raise GetBlobError("Error getting blob")
 
 
-async def upload_image(container_client, folder_uuid, image:str, image_uuid):
+async def upload_image(container_client, folder_name, image:str, image_uuid):
     """
     uploads the image to the specified folder within the user's container,
     if the specified folder doesnt exist, it creates it with a uuid
     
     Parameters:
     - container_client: the Azure container client
-    - folder_uuid: the name of the destination folder
+    - folder_name: the name of the destination folder
     - image:
     """
     try:
-        if not await is_a_folder(container_client, folder_uuid):
-            raise CreateDirectoryError(f"Folder:{folder_uuid} does not exist")
+        if not await is_a_folder(container_client, folder_name):
+            raise CreateDirectoryError(f"Folder:{folder_name} does not exist")
         else:
-            blob_name = "{}/{}.png".format(folder_uuid, image_uuid)
+            blob_name = "{}/{}.png".format(folder_name, image_uuid)
             metadata = {
                 "picture_uuid": f"{str(image_uuid)}",
-                "picture_set_uuid": f"{str(folder_uuid)}",
+                "picture_set_uuid": f"{str(folder_name)}",
```

Comment:
It doesnt sit right with me to have a tag for their uuid and the set value to their name...
```suggestion
                "picture_set_uuid": f"{str(folder_uuid)}",
```